specification.markdown

```diff
@@ -579,9 +579,9 @@ The maxver attribute is OPTIONAL. If the maxver attribute is not present then th
 
 ### \<documentation> Element
 
-Within an EBML Schema, the XPath of `<documentation>` element is `/EBMLSchema/element/documentation`.
+Within an EBML Schema, the XPath of `<documentation>` element are `/EBMLSchema/element/documentation` and `/EBMLSchema/element/restriction/enum/documentation`.
```

Comment:
If you're going the plural route, then "XPaths" instead of "XPath", I guess. And again "element" → "elements".

Reply:
I agree on this one. If there's an `are` it should be `elements`.

Reply:
Fixes the pluarl